VSharp.Explorer/AISearcher.fs

```diff
@@ -92,84 +119,121 @@ type internal AISearcher(oracle: Oracle, aiAgentTrainingOptions: Option<AIAgentT
         useDefaultSearcher <- stepsToSwitchToAI > 0u<step>
         q.Clear()
         availableStates.Clear()
+
     let update (parent, newSates) =
-        if useDefaultSearcher
-        then defaultSearcher.Update (parent,newSates)
-        newSates |> Seq.iter (availableStates.Add >> ignore)
+        if useDefaultSearcher then
+            defaultSearcher.Update(parent, newSates)
+
+        newSates
+        |> Seq.iter (availableStates.Add >> ignore)
+
     let remove state =
-        if useDefaultSearcher
-        then defaultSearcher.Remove state
+        if useDefaultSearcher then
+            defaultSearcher.Remove state
+
         let removed = availableStates.Remove state
-        assert removed       
-        for bb in state._history do bb.Key.AssociatedStates.Remove state |> ignore
-      
+        assert removed
+
+        for bb in state._history do
+            bb.Key.AssociatedStates.Remove state |> ignore
+
     let inTrainMode = aiAgentTrainingOptions.IsSome
-    
+
     let pick selector =
-        if useDefaultSearcher
-        then
+        if useDefaultSearcher then
             defaultSearcherSteps <- defaultSearcherSteps + 1u<step>
-            if Seq.length availableStates > 0
-            then
-                let gameStateDelta = collectGameStateDelta ()                
+
+            if Seq.length availableStates > 0 then
+                let gameStateDelta = collectGameStateDelta ()
                 updateGameState gameStateDelta
                 let statistics = computeStatistics gameState.Value
                 Application.applicationGraphDelta.Clear()
                 lastCollectedStatistics <- statistics
                 useDefaultSearcher <- defaultSearcherSteps < stepsToSwitchToAI
+
             defaultSearcher.Pick()
-        elif Seq.length availableStates = 0
-        then None
-        elif Seq.length availableStates = 1
-        then Some (Seq.head availableStates)
+        elif Seq.length availableStates = 0 then
```

Comment:
It is really ugly style.

Reply:
Fantomas has no way to move to the next line. Or I didn't search well.  
https://fsprojects.github.io/fantomas/docs/end-users/Configuration.html#Auxiliary-settings 

And yet, it seems that this style fits perfectly into the Microsoft manual: https://learn.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#multiline-bracket-formatting-styles.